### PR TITLE
feat: governance proposal list ui updated as per figma

### PR DESCRIPTION
**Pull request by ChiragPansuriya-iView**
governance proposal list
Screenshot 2024-02-20 at 11 39 50 AM

### CODE REVIEW

File: packages/components/GovernanceBox/GovernanceProgressBar.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          alignItems: "center",
+          backgroundColor: neutral33,
+          borderRadius: layout.spacing_x0_5,
+          flexDirection: "row",
+        }}
+      >
+        <View
+          style={{
+            width: percentageYes,
+            height: 4,
```
Comment (clegirar) [inline packages/components/GovernanceBox/GovernanceProgressBar.tsx:48]: Here maybe you can create a common style at the bottom of the file that have
```
height: 4,
borderRadius: 4,
zIndex: 2
```

File: yarn.lock [hunk 2]
```diff
@@ -20488,7 +20487,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"victory@npm:^36.6.12, victory@npm:^36.8.2":
+"victory@npm:^36.8.2":
```
Comment (clegirar) [inline yarn.lock:20490]: What is this ? Try to recover it and recup the main one if you don't use it

File: packages/utils/style/colors.ts [hunk 3]
```diff
@@ -43,7 +42,10 @@ export const neutralA3 = "#A3A3A3";
 export const transparentColor = "transparent";
 
 export const additionalRed = "#FFAEAE";
+export const additionalRed10 = "rgba(244, 111, 118, 0.1)";
```
Comment (clegirar) [inline packages/utils/style/colors.ts:45]: I know that in the codebase, we use `withAlpha` function directly in the code, so can you fit with that ?

**Review by clegirar — CHANGES_REQUESTED** [review]
Just little things, but in general it's good 👍 
I'll make a second review for the UI 👍

**Comment by clegirar** [comment]
image

For UI everything seems okay, the only thing that i reported is that the search bar at the top of the page (at left of period dropdown) doesn't worked, so please just make it works and everything will be okay 👍

File: yarn.lock (hunk 2 above)
Comment (clegirar) [reply yarn.lock:20500]: Yep so you can get the yarn.lock from main 👍

Comment (ChiragPansuriya-iView) [reply yarn.lock:20500]: Okay.

Comment (n0izn0iz) [reply yarn.lock]: if it's unused it's okay to remove, please do check that the other places where victory is used still work without the "victory" package though (nft details page for example)

**Review by clegirar — APPROVED** [review]
LGTM 👍

File: packages/screens/Governance/components/dropdowns/StatesDropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+import { SpacerColumn } from "@/components/spacer";
+import { useDropdowns } from "@/hooks/useDropdowns";
+import { neutral33, neutral55, secondaryColor } from "@/utils/style/colors";
+import { fontMedium14, fontSemibold14 } from "@/utils/style/fonts";
+import { layout } from "@/utils/style/layout";
+import { ProposalStatus } from "@/utils/types/gov";
+
+const def = {
+  all: {
+    name: "States in Period",
```
Comment (n0izn0iz) [inline packages/screens/Governance/components/dropdowns/StatesDropdown.tsx:19]:
```suggestion
    name: "All states",
```

File: packages/screens/Governance/components/dropdowns/StatesDropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+                );
+              })}
+            </View>
+          )}
+        </View>
+      </View>
+    </View>
+  );
+};
+const getKeys = Object.keys as <T extends object>(obj: T) => (keyof T)[];
```
Comment (n0izn0iz) [inline packages/screens/Governance/components/dropdowns/StatesDropdown.tsx:158]: use `objectKeys` from utils

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

File: packages/screens/Governance/components/dropdowns/StatesDropdown.tsx (hunk 5 above)
Comment (clegirar) [reply packages/screens/Governance/components/dropdowns/StatesDropdown.tsx:158]: Good to know

**Pull request merged**